### DISCUSSION TITLE
Configure number of jack output channels

### DISCUSSION THREAD

**Discussion by mitscherdinger** [FrancescoCeruti/linux-show-player#275] [Q&A] [answered]
Hello, again!

Cool, that LSP supports 8 output channels on jack, now! Using a 26-channel output card, I wonder if it would cause much trouble if the number of the output channels could be even more?

**Comment by FrancescoCeruti** [comment, marked as answer]
Hi,
the 8 outputs LiSP expose (on the left) are the audio-channels of the media file, the input (on the right) are your devices, there you should be able to see all yours 26-channels.

If can help, the Jack connection view works similar to the QtJackCtl on (is actually based on that).

**Comment by mitscherdinger** [comment]
Oh! Yes! Of course!

Somehow I was stuck in the ShowQ model looking at it. No need to change here anything at all. Close it, please!

Greets!
